### ISSUE TITLE
Custom CRUD UX open in a modal form

### ISSUE THREAD

**Issue by PhilippeAussel** [gustavnavar/Grid.Blazor#201]
Is it possible to use the custom crud feature to open the add or edit page in a modal window ? For example with Blazored.Modal ?

**Comment by gustavnavar** [comment]
No, it's not supported. CRUD pages are painted as full-screen ones

**Comment by PhilippeAussel** [comment]
> No, it's not supported. CRUD pages are painted as full-screen ones

Thank you for the reply, Are you interrested to add this feature ? I will plan to work on a fork for me but perhaps intreesting for the community ? Regards

**Comment by gustavnavar** [comment]
Yes, it's a nice feature to be included

**Comment by angelru** [comment]
any update?
